deploy/dynamo/sdk/src/dynamo/sdk/lib/service.py

```diff
@@ -224,14 +227,91 @@ def _remove_service_args(self, service_name: str):
                     }
                     os.environ["DYNAMO_SERVICE_ENVS"] = json.dumps(envs_config)
 
+    def inject_config(self) -> None:
+        """Inject configuration from environment into service configs.
+
+        This reads from DYNAMO_SERVICE_CONFIG environment variable and merges
```

Comment:
Can this be mentioned in a relevant README so users know to set it/how to set it/what it should contain?